code/game/machinery/computer/communications.dm

```diff
@@ -139,6 +139,7 @@ var/shuttle_call/shuttle_calls[0]
 					return
 				captain_announce(input)//This should really tell who is, IE HoP, CE, HoS, RD, Captain
 				log_say("[key_name(usr)] (@[usr.x],[usr.y],[usr.z]) has made a captain announcement: [input]")
+				world << sound('sound/misc/announce.ogg')

```

Comment:
This sends the sound to people on every zlevel, including the lobby and non station zlevels.